src/Microsoft.DotNet.ProjectModel/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.0.0-rc4-*",
+  "version": "1.0.0-featmsbuild-*",
```

Comment:
This doesn't seem right. ProjectModel RTM'd. Why do we have it at the same suffix as the preview components?